frontend/src/app/pages/validation-page/validation-page.component.ts

```diff
@@ -72,16 +72,9 @@ export class ValidationPageComponent implements OnInit {
         });
     }
 
-    private rect2DROIConverter(selections: any): Array<ROISelection2D> {
-        const roiSelections: Array<ROISelection2D> = [];
-        selections.forEach((selection: any) => {
-            roiSelections.push(new ROISelection2D(selection.x, selection.y, selection.slice_index, selection.width, selection.height));
-        });
-        return roiSelections;
-    }
-
     private requestSlicesWithLabel(): void {
-        this.labelService.getRandomLabel(this.rect2DROIConverter).then((label: Label) => {
+        // TODO: Unify with Marker Page...
```

Comment:
If this is a TODO, maybe we should create issue for that 💯

Reply:
This is Validation Page which will be refactored soon 😉